packages/core/src/components/collapse/collapse.tsx

```diff
@@ -230,3 +241,5 @@ export class Collapse extends AbstractPureComponent<ICollapseProps, ICollapseSta
         }
     }
 }
+
+polyfill(Collapse);
```

Comment:
It's probably safer to do the polyfill before exporting the component... side-effectful statements like this are code smells.

```ts
export const Collapse = polyfill(class extends AbstractPureComponent<...> {
```